core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1034,19 +1035,20 @@ class KafkaApis(val requestChannel: RequestChannel,
     var (authorizedTopics, unauthorizedForDescribeTopics) =
       topics.partition(topic => authorize(request.session, Describe, new Resource(Topic, topic)))
 
-    var unauthorizedForCreateTopics = Set[String]()
+    var authorizedForDescribeNotCreateTopics = Set[String]()
```

Comment:
Why did we rename this?

Reply:
the new name though longer and not pretty matches the what the set of topics actually is in terms of authorizations

Reply:
But then later we still have things like `unauthorizedForCreateTopicMetadata`. I think we should just keep this variable as it was since there is no actual change here. I'll push a minor commit.